docs/core_docs/docs/how_to/custom_tools.ipynb

```diff
@@ -26,6 +26,55 @@
     "The biggest difference here is that the first function requires an object with multiple input fields, while the second one only accepts an object with a single field. Some older agents only work with functions that require single inputs, so it's important to understand the distinction."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f6ec6ee8",
+   "metadata": {},
+   "source": [
+    "## `tool` function\n",
+    "\n",
+    ":::note\n",
+    "Only available in `@langchain/core` version 0.2.X and above.\n",
```

Comment:
nit: have been using lower case `x` to match `npm` style:

https://www.npmjs.com/package/semver

Reply:
Planning on replacing this before merge when I know the actual version number it'll be